file.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"log"
+	"code.cloudfoundry.org/bytefmt"
+)
+
+func BrowseDir(dirname string) error {
```

Comment:
Can you explain the concrete use case for this? It adds a dependency, so we need to be sure to really need it. If we need it, avoiding the dependency would be great.

Reply:
Thanks!